sdks/python/container/py2/build.gradle

```diff
@@ -51,6 +51,11 @@ task copyLauncherDependencies(type: Copy) {
   }
 }
 
+task copyDependencyLicenses(type: Copy) {
```

Comment:
Can this task be added to buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy in applyDockerNature?

Reply:
For Python, license files are written to docker image directly, so this task is no longer needed.
I will review Go and Java later when I work on them.,